src/pages/webinar/[webinarSlug]/index.tsx

```diff
@@ -52,7 +61,7 @@ const WebinarPage = ({
         link.download = `${userName}-webinar-certificate.png`;
         link.click();
       } catch (error) {
-        console.error('Error generatingcertificate image:', error);
+        console.error('Error generating certificate image:', error);
```

Comment:
Can you convert into a Hook -> useCertificate

Reply:
made the file useCertificate.ts in hooks but could not implement it